### PR TITLE
Update laptop-new.md

### PR DESCRIPTION
**Pull request by averyspeedybird**
Added some detail, made it more polished.

### CODE REVIEW

**Comment by nonkerdoob** [comment]
Add back {:target="_blank"}

they will not appear on the final website. It is used to tell the browser to open links in a new tab.

File: _guides/general/getting-started/laptop-new.md [hunk 1]
```diff
@@ -40,13 +41,15 @@ contributors: ['averyspeedybird', 'kirakenchin']
...
+- 3DMark TimeSpy (CPU, GPU and global performance scores)
+- Cinebench R23 (multi-core and single-core CPU performance)
+- CrystalDiskMark (config set to SSD) for SSD read/write performance
+- Built-in benchmark of the game Shadow of the Tomb Radier (free demo) with settings set to HIGHEST, once at native resolution and once at 1080p
 - No benchmark specific to RAM, as it is part of the global CPU performance
 
-## Conditions for Results Comparison  
+It is also very highly recommended to run a [combined load stress test](https://rentry.org/stresstest) to check your thermal performance out of the box.
+
+## Conditions for Result Submissions [to our discord](https://discord.gg/CtDvEHecHn)
```
Comment (nonkerdoob) [inline _guides/general/getting-started/laptop-new.md:52]: Add ``{:target="_blank"}``

File: _guides/general/getting-started/laptop-new.md [hunk 2]
```diff
@@ -71,10 +70,17 @@ Above conditions are only for comparison purpose, feel free to run bm with other
 | Power profile | performance Cooling | Back lifted |
 | Nvidia Driver | 511.23 |
 
+The above conditions are only for comparing your results to the other laptops on our discord server, feel free to run benchmarks with other parameters if you want to.
+
+## Extra
+Legion specific items: 
+
+- You could use [Legion Toolkit](https://github.com/BartoszCichecki/LenovoLegionToolkit/releases/tag/2.3.1) instead of vantage if you would like lower resource consumption
```
Comment (nonkerdoob) [inline _guides/general/getting-started/laptop-new.md]: Add ``{target="_blank"}``

File: _guides/general/getting-started/laptop-new.md (hunk 1 above)
Comment (averyspeedybird) [reply _guides/general/getting-started/laptop-new.md]: done that

**Review by nonkerdoob — APPROVED** [review]

**Pull request merged**